services/tables/src/main/java/com/linkedin/openhouse/tables/controller/TablesController.java

```diff
@@ -83,11 +83,8 @@ public ResponseEntity<GetTableResponseBody> getTable(
   public ResponseEntity<GetAllTablesResponseBody> getAllTables(
       @Parameter(description = "Database ID", required = true) @PathVariable String databaseId) {
 
-    com.linkedin.openhouse.common.api.spec.ApiResponse<GetAllTablesResponseBody> apiResponse =
-        tablesApiHandler.getAllTables(databaseId);
-
-    return new ResponseEntity<>(
-        apiResponse.getResponseBody(), apiResponse.getHttpHeaders(), apiResponse.getHttpStatus());
+    throw new UnsupportedOperationException(
+        "The getAllTables API is deprecated and please use searchTables API.");
```

Comment:
Let's remove the endpoint if its not being used anymore?

Reply:
There are a lot of things to remove if we want to remove this endpoint. Since we will add it back with paging support, do you think it's worth the work?